src/applications/appeals/995/containers/App.jsx

```diff
@@ -115,13 +94,6 @@ export const App = ({
             // resetStoredSubTask();
             setFormData({
               ...formData,
-              veteran: {
```

Comment:
so veteran data is included in platform profile?

Reply:
The `profileContactInfo` definition includes a [`wrapperKey` setting](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/forms-system/src/js/definitions/profileContactInfo.js#L67) that defaults to `veteran`. What that means is the platform code builds the form data object using the provided wrapper key and adds all the contact info data inside, e.g.

```js
// form data
veteran: {
  mailingAddress: { /* ... */ },
  mobilePhone: { /* ... */ },
  homePhone: { /* ... */ },
  email: '...',
}
```